docs/urban-airship.md

```diff
@@ -0,0 +1,47 @@
+# Urban Airship Integration
+
+Proximity Kit offers first class integration with the Urban Airship SDK. Proximity Kit levereges our meta data attributes to allow the integration settings to be configured remotely.
+
+This integration combines Proximity Regions with Urban Airship's Tags and Events.
+
+Note: This functionality is currently iOS only. Android coming soon, if you want to be updated when it is avaliable [let us know](mailto:support@radiusnetworks.com).
+
```

Comment:
Android _support is_ coming soon.  If you want to be updated when it is **available** (typo)...